### PR TITLE
String Array Utilities

### PR DESCRIPTION
**Pull request by bth1994**
At last

### CODE REVIEW

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 1]
```diff
@@ -42,31 +43,45 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+        return Arrays.asList(array).contains(value);
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]: Clean! 🚿 
Alternative could be to reuse your `getNumberOfOccurrences` method...
`return getNumberOfOccurrences(array, value) > 0`

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 2]
```diff
@@ -42,31 +43,45 @@ public static String getSecondToLastElement(String[] array) {
...
+        return reversed;
     }
 
     /**
      * @param array array of String objects
      * @return true if the order of the array is the same backwards and forwards
      */ // TODO
     public static boolean isPalindromic(String[] array) {
-        return false;
+        return Arrays.toString(array).equals(Arrays.toString(reverse(array)));
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]: `return Arrays.equals(array, reverse(array))`

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java [hunk 3]
```diff
@@ -84,23 +105,55 @@ public static int getNumberOfOccurrences(String[] array, String value) {
      * @return array with identical contents excluding values of `value`
      */ // TODO
     public static String[] removeValue(String[] array, String valueToRemove) {
-        return null;
+        List<String> list = new ArrayList<String>();
```
Comment (Git-Leon) [inline src/main/java/com/zipcodewilmington/StringArrayUtils.java]: 👍

File: src/main/java/com/zipcodewilmington/StringArrayUtils.java (hunk 2 above)
Comment (bth1994) [reply src/main/java/com/zipcodewilmington/StringArrayUtils.java]: Oh nice! that's a lot simpler. Didn't know arrays had that specific function

**Pull request closed without merging**